.changeset/mean-spiders-design.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/plugin-search-backend': minor
+---
+
+Be less restrictive with unknown keys on query endpoint by adding .passthrough() method to parse the `requestSchema`
```

Comment:
Along the same lines (consider the audience), you could probably remove `by adding .passthrough() method to parse the requestSchema` here too; it's probably not useful information in the changelog.